Resources/public/js/views/ez-languageselectionboxview.js

```diff
@@ -0,0 +1,412 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-languageselectionboxview', function (Y) {
+    "use strict";
+    /**
+     * Provides the language selection box view class
+     *
+     * @method ez-languageselectionboxview
+     */
+    Y.namespace('eZ');
+
+    var CONFIRM = 'confirm',
+        CANCEL = 'cancel',
+        BASE_TRANSLATION_AVAILABLE_CLASS = 'is-base-translation-allowed',
+        BASE_TRANSLATIONS_LIST_VISIBLE_CLASS = 'is-base-translations-list-visible',
+        TRANSLATION_HIGHLIGHT_CLASS = 'is-translation-selected';
+
+    /**
+     * The language selection box view.
+     *
+     * @namespace eZ
+     * @class LanguageSelectionBoxView
+     * @constructor
+     * @extends eZ.TemplateBasedView
+     */
+    Y.eZ.LanguageSelectionBoxView = Y.Base.create('languageSelectionBoxView', Y.eZ.TemplateBasedView, [], {
+        events: {
+            '.ez-languageselectionbox-close': {
+                'tap': '_cancel',
+            },
+            '.ez-languageselectionbox-confirm': {
+                'tap': '_confirm',
+            },
+            '.ez-new-translation': {
+                'tap': '_selectNewTranslation'
+            },
+            '.ez-base-translation': {
+                'tap': '_selectBaseTranslation'
+            },
+            '.ez-base-translation-checkbox': {
+                'change': '_toggleBaseTranslation'
+            },
+        },
+
+        initializer: function () {
+            this.on(['confirmHandlerChange', 'cancelHandlerChange'], function (e) {
+                this._syncEventHandler(e.attrName.replace(/Handler$/, ''), e.prevVal, e.newVal);
+            });
+            this._syncEventHandler('confirm');
+            this._syncEventHandler('cancel');
+            this.after('activeChange', function () {
+                if ( this.get('active') ) {
+                    this._uiUpdateTitle();
+                }
+            });
+            this._publishEvents();
+            this.on('existingTranslationsChange', function (e) {
```

Comment:
it seems it's because availableTranslations is set after the newTranslations attribute. We can of course hack the order of the properties in the return value of `_getViewParameters` but that's a bit dangerous. Instead, I would do a render when is becoming active (so you can remove the `_uiUpdateTitle` thing which was introduce because others side views like that don't need to be rerendered)